src/badguy/badguy.hpp

```diff
@@ -73,10 +74,19 @@ class BadGuy : public MovingSprite,
 
   /** True if this badguy can break bricks or open bonusblocks in his
       current form. */
-  virtual bool can_break() const { return false; }
+  virtual bool can_break() const
+  {
+    return false;
+  }
 
-  Vector get_start_position() const { return m_start_position; }
-  void set_start_position(const Vector& vec) { m_start_position = vec; }
+  Vector get_start_position() const
+  {
+    return m_start_position;
+  }
+  void set_start_position(const Vector& vec)
+  {
+    m_start_position = vec;
+  }
 
```

Comment:
Everything on one line (one line per function, including their bodies and brackets).

Functions are only defined in the header if they are trivial (like, you could recite the exact contents of their body simply by looking at the function's name, often `get_some_member_variable()` or `set_some_member_variable()`). IIRC, they're defined in the header to avoid cluttering the body file.

Functions in the header will only span multiple lines if they are trivial but ridiculously long for some reason (like the `expose` and `unexpose` functions of the Will-o-Wisp and the dispenser)

Reply:
Fixed